### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by matan-mitiga**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
```
Comment (schachar) [inline .gitignore:1]: there is a pre-set node ignore list from github maybe take it

File: src/index.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import fastify from 'fastify'
```
Comment (schachar) [inline src/index.ts:1]: this will not encorage use of DI or types
is this what we want?

File: src/services/HelloWorldService.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+export class HelloWorldService {
+  public getHelloWorldString() {
```
Comment (schachar) [inline src/services/HelloWorldService.ts]: return void, or promise of void

Comment (schachar) [reply src/services/HelloWorldService.ts]: actually return string or promise of string :)

**Comment by schachar** [comment]
small comments regarding return type missing
also, do we want eslint but with disabled config that the candidate can enable?
not critical

**Review by schachar — APPROVED** [review]

File: src/index.ts (hunk 2 above)
Comment (matan-mitiga) [reply src/index.ts:1]: why? isn't this considered some kind of DI? 

```
const helloWorldController = new HelloWorldController(new HelloWorldService())
```

File: .gitignore (hunk 1 above)
Comment (matan-mitiga) [reply .gitignore:1]: These are sufficient, the candidate may add some if he wants

**Pull request merged**